nav2_docking/opennav_docking_core/include/opennav_docking_core/non_charging_dock.hpp

```diff
@@ -129,6 +129,13 @@ class NonChargingDock : public ChargingDock
    * @brief Gets if this is a charging-typed dock
    */
   bool isCharger() final {return false;}
+
+  /**
+   * @brief Indicates the direction of the dock. This is used to determine if the
+   * robot should drive forwards or backwards onto the dock.
+   * @return DockDirection The direction of the dock
+   */
+  virtual DockDirection getDockDirection() = 0;
```

Comment:
Why not default to forward as well, as ChargingDock does? I would think we should at least make them consistent, one way or another